AI Linear Activation Function (Issue #2700) (#3036)
Codecademy/docs: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md

---
Title: 'Linear Activation Function'
Description: 'The linear activation function, also known as the identity function, is where the activation is proportional to the input.'
Subjects:
  - 'Machine Learning'
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'AI'
  - 'Machine Learning'
  - 'Neural Networks'
  - 'Functions'
CatalogContent:
  - 'machine-learning'
  - 'path/data-science'
---

The **linear activation function**, also known as the identity function, is where the activation is proportional to the input. The function doesn't do anything to the weighted sum of the input, it simply spits out the value it was given.

Mathematically, it can be defined as:

> ![linear Activation Function Equation](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation.png)

Reviewer: This link yields a 404 error

---
Title: 'Linear Activation Function'

> Description: 'The linear activation function also known as identity function, where the activation is proportional to the input.'

Reviewer 2: ```suggestion
Description: 'The linear activation function, also known as the identity function, is where the activation is proportional to the input.'
```

Subjects:
  - 'Machine Learning'
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'AI'
  - 'Machine Learning'
  - 'Neural Networks'
  - 'Functions'
CatalogContent:
  - 'machine-learning'
  - 'path/data-science'
---

> The **linear activation function** is also known as **straight-line function** where the activation is proportional to the input.The function doesn't do anything to the weighted sum of the input, it simply spits out the value it was given.

Reviewer 2: ```suggestion
The **linear activation function**, also known as the identity function, is where the activation is proportional to the input. The function doesn't do anything to the weighted sum of the input, it simply spits out the value it was given.
```

Mathematically, it can be defined as:

![linear Activation Function Equation](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation.png)

The value of variable a can be any constant value

> chosen by the user.

Reviewer 2: There's no variable "a" in the above equation.

### Usage and Limitations

> The problem with this activation is that it cannot be defined in a specific range. Applying this function in all the nodes makes the activation function work like linear regression.

Reviewer 2: ```suggestion
The problem with this activation function is that it cannot be defined for a specific range. Applying this function in all the nodes makes the activation function work like linear regression.
```

---
Title: 'Linear Activation Function'
Description: 'The linear activation function, also known as the identity function, is where the activation is proportional to the input.'
Subjects:
  - 'Machine Learning'
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'AI'
  - 'Machine Learning'
  - 'Neural Networks'
  - 'Functions'
CatalogContent:
  - 'machine-learning'
  - 'path/data-science'****
---

> The **linear activation function**, also known as the identity function, is one of the most straight-forward activation function,where the output of linear activation function is exactly same as input.

Reviewer: ```suggestion
The **linear activation function**, also known as the identity function, is a straight-forward activation functions where the output is exactly same as the input.
```

> In general , linear activation function simply add weighted sum with bias and pass it as output.

Reviewer: This statement contains some grammatical errors and is unclear. Can you reword it?

Mathematically, it can be defined as:

![linear Activation Function Equation](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation.png)

where :

![linear Activation Function](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation-express.png)

### Usage and Limitations

The problem with this activation function is that it is limited in the context of deep neural network's.

> Using linear activation functions would not able to the model to learn complex non-linear relationships between input's and output's.

Reviewer: This statement needs to be reworded.
